demo/demo_secondary.py

```diff
@@ -63,6 +63,7 @@
 _ecu_serial = 'TCUdemocar'
 _primary_host = demo.PRIMARY_SERVER_HOST
 _primary_port = demo.PRIMARY_SERVER_DEFAULT_PORT
+_partial_verifying = False
```

Comment:
Have you tried running the [demo](https://github.com/uptane/uptane/#1-starting-the-demo) with `_partial_verifying = True`?

Reply:
Yes, I did!